harvesting_metadata_configs.md

```diff
@@ -0,0 +1,39 @@
+# Harvesting metadata configuration Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/harvesting_metadata_configs**   
```

Comment:
I think we need to better align these two endpoint names.
* `/api/config/harvesting_metadata_configs`
* `/api/core/collections/<:uuid>/harvestingSettings`

I'd like to make these more similar. Currently one is camelcase, while the other uses underscores. One uses the term "Settings" while the other uses "Configs".

Maybe these should be called:
* `/api/config/harvestingMetadataConfig`
* `/api/core/collections/<:uuid>/harvestingConfig` or just `/api/core/collections/<:uuid>/harvesting`

Other names are fine too, I just would like to see a more consistent name here.

Reply:
I agree with Tim, if we look at other /config/* endpoints, we will have:
- /api/config/submissiondefinitions
- /api/config/submissionsections
- /api/config/submissionforms
- /api/config/submissionuploads

I think it should be coherent with what was defined. And based on what is already defined, my suggestion goes to rename it:
from:
`/api/config/harvesting_metadata_configs`
to:
`/api/config/harvestingmetadataconfigs`

**Note**: initially I was wondering if the name make sense or should be under collection (like: collectionharvestingdefinitions), since this only applies to collections. But that name could be ambiguous. 

Or perhaps if don't agree with the format, we should revisit the old ones if we plan to use a different format.

Reply:
Thinking on this a bit more (and based on @paulo-graca's notes), as the word `/config` is already in the path, I'd like to recommend we consider shortening these long path names to something like:
* `/api/config/harvestermetadata`
* `/api/core/collections/<:uuid>/harvester`

I'm suggesting to use the term "harvester" as it's a noun (instead of "harvesting" which is an action verb). Plus it's the same term used in the backend configuration: https://wiki.duraspace.org/display/DSDOC6x/OAI#OAI-OAI-PMH/OAI-OREHarvester(Client)

I've also dropped "config" or "settings" from these paths, as none of our other configurable based endpoints append those terms...they are essentially implied.

Reply:
I've update the PR based on the latter proposal, assuming this is ok for everyone